R/analyse_SAR.CWOSL.R

```diff
@@ -1553,7 +1683,8 @@ if(is.list(object)){
     # Set return Values -----------------------------------------------------------
 
     ##generate unique identifier
-    UID <- create_UID()
```

Comment:
Please restore the original code here, accessing objects via `package:::` is not possible from within the package and violates the CRAN rules.